apps/desktop/src/lib/stack/CurrentSeries.svelte

```diff
@@ -39,9 +39,25 @@
 	);
 	const forgeChecksMonitorStore = createForgeChecksMonitorStore(undefined);
 	$effect(() => forgeChecksMonitorStore.set(checksMonitor));
+
+	const seriesType = currentSeries.patches[0] ? currentSeries.patches[0].status : 'local';
```

Comment:
I have a helper for this actually since we need to look it up in a few places already. See: `apps/desktop/src/lib/branch/stackingUtils.ts`

Reply:
Thanks! Updated with your function